### ISSUE TITLE
Typeform logo not found in /apps/typeform/how-to-use

### ISSUE THREAD

**Issue by ShashankGupta10** [calcom/cal.com#18390]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

Typeform logo 404  in /apps/typeform/how-to-use
Alt text also wrong

### Steps to Reproduce

1. Go to https://app.cal.com/apps/typeform/how-to-use

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?
Image should be there, It is not.

### Actual Results

- Image not there

### Expected Results

- Image should be there also alt text should be there

### Technical details

- Prod
![image](https://github.com/user-attachments/assets/8e47df96-2fa3-45af-872a-a6e371cd48eb)

### Evidence

- Visible on hosted platform
